models/cex/algorand/cex_algorand_addresses.sql

```diff
@@ -0,0 +1,23 @@
+{{config(
+        tags = ['static'],
+        alias = 'addresses',
+        post_hook='{{ expose_spells(\'["algorand"]\',
```

Comment:
do we want all of them exposed on the data explorer?
just double checking. should be fine if so.

Reply:
Yes, having all of them exposed is perfect imo